src/components/Contentful/DynamicRichText.vue

```diff
@@ -52,16 +50,7 @@ export default {
 		}
 	},
 	render() {
-		return h(
-			'div', /* Wrap output in a div since Vue templates must have 1 root element */
-			[
-				h(this.dynamicComponent, {
-					class: this.elementClasses,
-					props: {
-					},
-				})
-			]
-		);
+		return h(this.dynamicComponent, { class: this.elementClasses });
```

Comment:
Do we need to blend in props here or is that automatic?

Reply:
Should just work